src/main/java/teammates/ui/controller/StudentCommentsPageAction.java

```diff
@@ -106,57 +98,44 @@ private String getCoursePaginationList(List<String> coursePaginationList)
         String courseName = "";
         List<CourseAttributes> courses = logic.getCoursesForStudentAccount(account.googleId);
         java.util.Collections.sort(courses);
-        for(int i = 0; i < courses.size(); i++){
+        for (int i = 0; i < courses.size(); i++) {
             CourseAttributes course = courses.get(i);
             coursePaginationList.add(course.id);
-            if(courseId == ""){
+            if (courseId == "") {
```

Comment:
This is changing the previous logic, but we should probably correct this mistake anyway.